mini-e2e-helm.groovy

```diff
@@ -21,6 +22,21 @@ node('cico-workspace') {
 			changelog: false
 	}
 
+	stage('skip ci/skip/e2e label') {
+		if (params.ghprbPullId == null) {
+			skip_e2e = 1
+			return
+		}
+		skip_e2e = sh(
+			script: "./scripts/get_github_labels.py --id=${ghprbPullId} --has-label=ci/skip/e2e",
+			returnStatus: true)
+	}
+	// if skip_e2e returned 0, do not run full tests
+	if (skip_e2e == 0) {
```

Comment:
this should be skip_e2e==1 isnt it?

Reply:
function get_github_labels.py returns 0 in case the label is found this should be fine.